image/registry_handler_test.go

```diff
@@ -0,0 +1,35 @@
+package image
+
+import (
+	"testing"
+
+	"github.com/sclevine/spec"
+	"github.com/sclevine/spec/report"
+
+	h "github.com/buildpacks/lifecycle/testhelpers"
+)
+
+func TestRegistryHandler(t *testing.T) {
+	spec.Run(t, "RegistryHandler", testRegistryHandler, spec.Parallel(), spec.Report(report.Terminal{}))
+}
+func testRegistryHandler(t *testing.T, when spec.G, it spec.S) {
+	when("insecure registry", func() {
+		it("returns WithRegistrySetting options for the domains specified", func() {
```

Comment:
Can we get a test added that returns more than one?

Reply:
Sure thing!